main.go

```diff
@@ -512,9 +513,14 @@ func main() {
 		trustedPkLut[pair.String()] = true
 	}
 
+	c := cors.New(cors.Options{
+		AllowedOrigins: []string{"*"},
+		AllowedMethods: []string{http.MethodGet, http.MethodPost},
```

Comment:
Did you test it? I would expect that you'd also need `OPTIONS`.

Reply:
(Or does that library add it by default?)

Reply:
I did test it; locally this works fine. (And indeed [the library adds it by default](https://github.com/go-chi/cors/blob/master/cors.go#L368).)